.gitignore

```diff
@@ -24,4 +24,5 @@ xcshareddata/
 *.xccheckout
 *.moved-aside
 *.xcuserstate
-.vscode/
\ No newline at end of file
+.vscode/
+subprojects/*/
```

Comment:
why are these ignored?

Reply:
i was wondering about subprojects because i thought this would match the `.wrap` files, but i must have missed the trailing slash. all good!